package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "start": "node ./src/bin/www",
+    "start": "set DEBUG=dev && node ./src/bin/www",
```

Comment:
This would be our start script for production, we should not set debug to true in production. I think you should add it to the 'devstart' script rather but the babel watch takes care of that

Reply:
Thanks for your feedback. babel watch and debug are different. debug logs the string output of `Listening on port 3000` to the console to replace console.log that eslint was complaining about. If I remove debug from start scrip, you won't be able to see the log of that string when the server starts running.

It is also on the start script cause that's the script that starts the server.

Reply:
> Thanks for your feedback. babel watch and debug are different. debug logs the string output of `Listening on port 3000` to the console to replace console.log that eslint was complaining about. If I remove debug from start scrip, you won't be able to see the log of that string when the server starts running.
> 
> It is also on the start script cause that's the script that starts the server.

Thats great, good job